### PR TITLE
chore(button): add known issue for tooltip/popover

### PR DESCRIPTION
**Pull request by icfantv**
* add known issue to button docs indicating a CSS issue when using a tooltip or
  a popover.

### CODE REVIEW

File: src/buttons/docs/readme.md [hunk 1]
```diff
@@ -44,3 +44,7 @@ With the buttons directive, we can make a group of buttons behave like a set of
 * `toggleEvent`
   _(Default: `click`)_ -
   Event used to toggle the buttons.
+
+### Known issues
+
+To use tooltips or popovers on a button group, make sure you add the applicable tooltip or popover directive directly to your `btn-group` element and set `appendToBody` to `true`. This is due to Bootstrap CSS styling. See [here](http://getbootstrap.com/components/#btn-groups) for more information.
```
Comment (RobJacobs) [inline src/buttons/docs/readme.md]: The append-to-body is only necessary if adding the tooltip to an element inside the btn-group.

Comment (icfantv) [inline src/buttons/docs/readme.md]: Ah, ok.  I wasn't sure I understood this correctly.

**Comment by icfantv** [comment]
@RobJacobs, fixed.  look good?

**Comment by RobJacobs** [comment]
@icfantv I would remove the mention of adding the tooltip to the btn-group, so maybe something like:

make sure you set `appendToBody` to `true`

**Comment by icfantv** [comment]
@RobJacobs ok, done.

**Pull request closed without merging**